buf/registry/module/v1beta1/label_service.proto

```diff
@@ -157,12 +164,26 @@ message ListLabelHistoryRequest {
 }
 
 message ListLabelHistoryResponse {
+  message Value {
+    // The Commit.
+    Commit commit = 1 [(buf.validate.field).required = true];
+
+    // The Label's review state.
+    //
+    // This field will only be set if governance is enabled for the Label.
```

Comment:
Think the docs here should clarify that this is the state of the latest commit for that label.

Reply:
Is this not the review state of this particular Commit in this history of this Label? If so, it is odd to use `LabelReviewState` here, the docs on that reference "last commit". I would expect a nested enum like `CommitReviewState`.

Reply:
Yeah, I agree with Saquib here.

I pushed a commit to rename `LabelReviewState` to `CommitReviewState`.